build.gradle.kts

```
@@ -5,11 +5,11 @@ import org.jetbrains.kotlin.utils.addToStdlib.safeAs
 
 plugins {
     kotlin("jvm") version "1.3.72" apply false
-    id("com.google.protobuf") version "0.8.15" apply false
+    id("com.google.protobuf") version "0.8.17" apply false
 }
 
-ext["grpcVersion"] = "1.36.0" // CURRENT_GRPC_VERSION
-ext["protobufVersion"] = "3.14.0"
+ext["grpcVersion"] = "1.41.0" // CURRENT_GRPC_VERSION
+ext["protobufVersion"] = "3.18.0"
```

Not compatible with `3.18.0` yet.

```suggestion
ext["protobufVersion"] = "3.17.3"
```